grails-app/controllers/au/org/ala/biocollect/BioActivityController.groovy

```diff
@@ -2242,7 +2243,7 @@ class BioActivityController {
     }
 
     private String getMessage(Map resp) {
-        def errorMessage
+        Map errorMessage
```

Comment:
It is expecting Map here. but in line 2251 and 2255, it returns `resp.error`. I believe that is a string. Also, the signature says the method returns String `private String getMessage(Map resp) {`. For consistency, make sure this method returns Map.